common/djangoapps/student/models.py

```diff
@@ -2066,10 +2066,14 @@ def add_to_profile_url(self, course_key, course_name, cert_mode, cert_url, sourc
 
     def _cert_name(self, course_name, cert_mode):
         """Name of the certification, for display on LinkedIn. """
```

Comment:
This function is unseemingly complex in that it references another field, calls into potentially-overridden settings, makes assumptions about the schema in order to pull the proper info, and depends on specific formatting to render the correct output.  We don't necessarily have to change all that, but we should at least explain what is going on in the docstring.